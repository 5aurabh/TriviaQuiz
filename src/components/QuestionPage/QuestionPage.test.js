import React from 'react';
import renderer from 'react-test-renderer';
import Questionpage from '.';

it('Questionpage: default', () => {
  const component = renderer.create(<Questionpage questions={[]} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
