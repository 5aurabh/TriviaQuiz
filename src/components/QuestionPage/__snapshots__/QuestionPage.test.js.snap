// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Questionpage: default 1`] = `
<div
  className={undefined}
>
  edit me: at src/components/QuestionPage/index.js
  <div
    className={undefined}
  >
    <div
      className={undefined}
      onClick={[Function]}
    >
      <span
        className={undefined}
      >
        <svg
          attr={
            Object {
              "viewBox": "0 0 512 512",
            }
          }
          className={undefined}
          fill="currentColor"
          height="1.5em"
          size="1.5em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 512 512"
          width="1.5em"
        >
          <path
            d="M401.4 224h-214l83-79.4c11.9-12.5 11.9-32.7 0-45.2s-31.2-12.5-43.2 0L89 233.4c-6 5.8-9 13.7-9 22.4v.4c0 8.7 3 16.6 9 22.4l138.1 134c12 12.5 31.3 12.5 43.2 0 11.9-12.5 11.9-32.7 0-45.2l-83-79.4h214c16.9 0 30.6-14.3 30.6-32 .1-18-13.6-32-30.5-32z"
          />
        </svg>
      </span>
    </div>
    <div
      className={undefined}
      onClick={[Function]}
    >
      <span
        className={undefined}
      >
        <svg
          attr={
            Object {
              "viewBox": "0 0 512 512",
            }
          }
          className={undefined}
          fill="currentColor"
          height="1.5em"
          size="1.5em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 512 512"
          width="1.5em"
        >
          <path
            d="M284.9 412.6l138.1-134c6-5.8 9-13.7 9-22.4v-.4c0-8.7-3-16.6-9-22.4l-138.1-134c-12-12.5-31.3-12.5-43.2 0-11.9 12.5-11.9 32.7 0 45.2l83 79.4h-214c-17 0-30.7 14.3-30.7 32 0 18 13.7 32 30.6 32h214l-83 79.4c-11.9 12.5-11.9 32.7 0 45.2 12 12.5 31.3 12.5 43.3 0z"
          />
        </svg>
      </span>
    </div>
  </div>
  <div
    className={undefined}
  >
    <button
      className={undefined}
      disabled={false}
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#D41602",
          "color": "#fff",
          "cursor": "pointer",
        }
      }
    >
      Abort
    </button>
  </div>
</div>
`;
